### ISSUE TITLE
iOS/iPadOS App plans?

### ISSUE THREAD

**Issue by antiero** [FancyPixel/moby#2]
Hey @andreamazz - amazing tool! Do you have any plans to produce a similar timeline tool to your web timeline, as a native App for iPad/iPhone/macOS? I've been thinking about making one in SwiftUI and came across yours... would be happy to collaborate if a native App isn't in your plans!? I think it would be super handy for testing to play and edit haptics on the device in situ!

**Comment by andreamazz** [comment]
Hey @antiero 
I thought about making an app at first, but it would only make sense on iPhone since it's the only device with haptic feedback. Since the timeline manipulation requires precision, I opted for a web version instead. Even if it adds an extra step (i.e. refreshing the file on the device once edited) the extra screen estate and the precision of the mouse pointer won me over.
If you have any ideas for a more usable touch UI I'd be happy to take a look at it though 👍

**Comment by antiero** [comment]
Yeah, makes sense about it only being available on iPhone.. My thoughts were that if haptics were unavailable on the device, we could play back some low frequency audio and animation, similar to Apple's recent Core Haptics WWDC 2019 talk, where they had to engage the audience with haptics, without anyone able to actually feel them in the audience!

**Comment by andreamazz** [comment]
That's a good idea 👍 
Having an editor on iPad would, in fact, be really cool.
